source/security/private-endpoints.txt

```diff
@@ -215,20 +290,18 @@ connection using either the App Services UI or the Admin API.
                  -d '{
                    "cloud_provider_region": "<your-cloud-deployment-region>",
                    "cloud_provider_endpoint_id": "<your-vpc-endpoint-id>"
+                   "endpoint_type":"sync"
                  }'
+            
+            #. You must also create a General Private Endpoint by calling
+               the ``Create a VPC Private Endpoint`` Admin API endpoint again.
+               This time, set the ``endpoint_type`` to ``"general``.
 
-         .. step:: Modify Your VPC Private DNS
-
-            Use the AWS CLI to modify your VPC private DNS settings. This
-            ensures that your App can resolve the private endpoint's DNS name
-            to the correct IP address.
-
-            .. code-block:: sh
+Migrate a Legacy Endpoint
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-               aws ec2 modify-vpc-endpoint \
-                 --region <your-aws-deployment-region> \
-                 --vpc-endpoint-id <your-vpc-endpoint-id> \
-                 --private-dns-enabled
+To migrate a Legacy Endpoint to a Sync or General endpoint, you must create
+new endpoints that use your existing VPC endpoints.
```

Comment:
> that use your existing VPC endpoints

Is this true? Do we want people using their existing VPC endpoints? Or should we encourage them to create new VPC endpoints too?

Any other considerations for migrating?